### PR TITLE
Addition of .pc file to autotools process

### PR DESCRIPTION
**Pull request by jkordani**
I'm not sure this is complete yet, but I was having a hard time figuring out everything that the cmake process brings in to make the file.  Also, I'm not sure where to go so that this file is installed appropriately during make install

### CODE REVIEW

File: config/hdf5.pc.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: hdf5
+Description: HDF5 (Hierarchical Data Format 5) Software Library
+Version: 1.13.1-1
```
Comment (byrnHDF) [inline config/hdf5.pc.in:8]: This version should be a @var@ instead of hard-coded.

**Comment by byrnHDF** [comment]
Check out the bin/h5cc.in file for consideration of variables.
Also, the bin/h5cc.in file includes the HL libraries.

A search on h5cc i the source should indicate installation commands, which could be used as a template.

**Review by byrnHDF — CHANGES_REQUESTED** [review]

File: config/hdf5.pc.in (hunk 1 above)
Comment (byrnHDF) [reply config/hdf5.pc.in:8]: See src/libhdf5.settings.in for examples.

**Comment by derobins** [comment]
There has been no movement on this in a long time. We'll attempt to add pkg-config support to the Autotools for 1.12.3 via a separate PR.

**Pull request closed without merging**